grammars/javascript.cson

```diff
@@ -354,8 +354,12 @@
     'name': 'entity.name.type.object.js.console'
   }
   {
-    'match': '\\.(warn|info|log|error|time|timeEnd|assert)\\b'
-    'name': 'support.function.js.console'
+    'match': '\\b(\\.)(assert|clear|debug|error|info|log|profile|profileEnd|time|timeEnd|warn)\\b(?=\\()'
```

Comment:
The first one is definitely required to prevent `console,.log()` from matching.  It doesn't look like the second one is needed though; I'll remove that.